TinyOS/main.c

```diff
@@ -26,16 +27,25 @@ void LCD_Write_Command(void);
 void LCD_Read_Data(void);
 void Mega328P_Init(void);
 void ADC_Get(void);
+void ADC_Poll(void);
 void EEPROM_Read(void);
 void EEPROM_Write(void);
+void UART_Poll(void);
+void substring(char*, char*, int);
+void scrollingLCD(char*);
+void LCD_Delay(void);
 
 unsigned char ASCII;			//shared I/O variable with Assembly
 unsigned char DATA;				//shared internal variable with Assembly
 char HADC;						//shared ADC variable with Assembly
 char LADC;						//shared ADC variable with Assembly
 
-char volts[5];					//string buffer for ADC output
+char lastTemperature[5] = "     ";
```

Comment:
Not sure what this is doing, empty space?

Reply:
It's used to store the previous temperature. The display is only updated when there is a difference between the last temperature and the current temperature, and then the last temperature is updated.